Week-3/2_CheckAccountType/Readme.md

```diff
@@ -0,0 +1,12 @@
+# Write a small solidity function that - 
+
+1. Accepts 1 address as argument 
+2. Returns boolean False if that address is an EOA 
+3. Returns TRUE if that address is a Contract
+
+## Brief
```

Comment:
You can add a line about difference between EOA and contract address

Reply:
Done